common/religion/religions/necromancy_group.txt

```diff
@@ -24,6 +24,20 @@
 	doctrine = doctrine_deviancy_accepted
 	doctrine = doctrine_witchcraft_accepted
 
+	#Magic
+	doctrine = doctrine_light_magic_crime
+	doctrine = doctrine_shadow_magic_ignored
```

Comment:
Necromancy uses a lot of shadow magic, should be accepted.

Reply:
Done